src/main/kotlin/no/nav/syfo/behandlerdialog/kafka/KafkaMeldingFraBehandler.kt

```diff
@@ -78,12 +84,33 @@ class KafkaMeldingFraBehandler(private val database: DatabaseInterface) : KafkaC
         connection: Connection,
     ) {
         log.info("Received melding fra behandler with uuid: ${melding.referanseUuid}")
+        val existingOppgave = connection
+            .getPersonOppgaveByReferanseUuid(melding.referanseUuid)
+            ?.toPersonOppgave()
+        if (existingOppgave != null && existingOppgave.type == PersonOppgaveType.BEHANDLERDIALOG_MELDING_UBESVART) {
```

Comment:
Hva om existingOppgave allerede er behandlet?

Reply:
Hmm, godt poeng 🤔 Det kan jo være caser der det ligger en behandlet oppgave der, så burde ta høyde for det ja 👍🏼